contracts/LiquidationsPeripheral.vy

```diff
@@ -798,16 +805,19 @@ def liquidateNFTX(_collateralAddress: address, _tokenId: uint256):
         _tokenId
     )
 
-    IERC721(_collateralAddress).transferFrom(
-        ICollateralVaultPeripheral(self.collateralVaultPeripheralAddress).collateralVaultCoreAddress(),
-        self,
-        _tokenId
-    )
+    ICollateralVaultPeripheral(self.collateralVaultPeripheralAddress).transferCollateralFromLiquidation(self, _collateralAddress, _tokenId)
```

Comment:
The removal of the `IERC721.transferFrom` and the use of the `ICollateralVaultPeripheral.transferCollateralFromLiquidation` is a very nice catch!

This means that the `ICollateralVaultPeripheral.approveBackstopBuyer` above can be removed, right?